internal/pkg/workspace/workspace.go

```diff
@@ -515,26 +522,62 @@ func (ws *Workspace) Path() (string, error) {
 	return filepath.Dir(copilotDirPath), nil
 }
 
-// Rel returns the path relative to the workspace root.
-func (ws *Workspace) Rel(fullPath string) (string, error) {
+// RelWsRoot returns the path relative to the workspace root.
+// The input path is allowed to be either relative to the current
+// working directory or absolute.
+//
+// This is useful for storing a file path in configuration.
+//
+// Prefer RelCwd for displaying paths to users.
+func (ws *Workspace) RelWsRoot(path string) (string, error) {
 	copiDir, err := ws.copilotDirPath()
 	if err != nil {
 		return "", fmt.Errorf("get path to Copilot dir: %w", err)
 	}
+	fullPath, err := filepath.Abs(path)
+	if err != nil {
+		return "", fmt.Errorf("make path absolute: %w", err)
+	}
 	return filepath.Rel(filepath.Dir(copiDir), fullPath)
 }
 
+// RelCwd returns the path relative to the current working directory.
+// The input path is allowed to be either relative to the current
+// working directory or absolute.
+//
+// This is useful when displaying a file path to the user.
+//
+// You probably want to use RelWsRoot for most other cases.
+// In the common case (the user running commands from the workspace root)
+// these functions have the same behavior, which can be a dangerous
+// source of false assumptions because that is not guaranteed!
+func (ws *Workspace) RelCwd(fullPath string) (string, error) {
```

Comment:
At a glance, I can't tell why this needs to belong to the `Workspace` object 💭 I understand `Rel` because it's relative to the workspace root but this could be also written as a function in the cli pkg:
```go
package cli

func relCwd(path string) (string, error) {
   if !filepath.IsAbs(path) { return filepath.Clean(path), nil }
   wd, err := os.Getwd()
   if err != nil {
      return "", err
   }
   return filepath.Rel(wd, path), nil
}
```

to make the above easy to unit test in commands:
```go
// in cmd.go
package cli 

func relCwd(getWd() (string, error), path string) (string, error) { ... }
type Cmd struct {
   getWd func() (string, error)
}
func NewCmd() *Cmd {
   return &Cmd{ getWd: os.Getwd }
}

// in cmd_test.go
testCmd := &Cmd{
  getWd: func() { return "root", nil }
}
```